public/newhaven.php

```diff
@@ -112,27 +112,27 @@
   <div class="ui vertical stripe segment" id="register">
     <div class="ui text container">
       <h3 class="ui header">Registration TBA</h3>
-<!--      <tito-widget event="indiewebcamp/baltimore-2018"><a href="https://ti.to/indiewebcamp/baltimore-2018">Get Tickets</a></tito-widget>-->
+      <tito-widget event="indiewebcamp/baltimore-2018"><a href="https://ti.to/indiewebcamp/baltimore-2018">Get Tickets</a></tito-widget>
 
     </div>
   </div>
 
 
-<!--  <div class="ui vertical stripe segment" id="rsvps">-->
-<!--    <div class="ui text container">-->
-<!--      <h3 class="ui header">Indie RSVPs</h3>-->
+  <div class="ui vertical stripe segment" id="rsvps">
+    <div class="ui text container">
+      <h3 class="ui header">Indie RSVPs</h3>
 
-<!--      <p>See <a href="https://indieweb.org/RSVP">indieweb.org/RSVP</a> for instructions on how to create an RSVP post. Once you've created the RSVP post which links to this page, send a Webmention and you'll appear below!</p>-->
+      <p>See <a href="https://indieweb.org/RSVP">indieweb.org/RSVP</a> for instructions on how to create an RSVP post. Once you've created the RSVP post which links to this page, send a Webmention and you'll appear below!</p>
 
         <?php //include('../templates/show-rsvps.php'); ?>
```

Comment:
Should probably uncomment this to show RSVPs via Webmention

Reply:
uncommented! also uncommented display of ti.to RSVPs.